src/Surfnet/ServiceProviderDashboard/Domain/Entity/Entity.php

```diff
@@ -40,27 +40,6 @@
  */
 class Entity
 {
-    const BINDING_HTTP_POST = 'urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST';
```

Comment:
By moving the constants out of this class the usage of `self::` in this class wouldn't work anymore. Please resolve that.